BitmarkRegistry/Helpers/NetworkManager.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ReachabilityManager.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/10/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import Reachability
+
+struct NetworkManager {
```

Comment:
Consider not to make a class / manager to do this lightweight task.
In fact, `NetworkManager` seems to contain more logic than just reachability.